sbt-plugin/src/sbt-test/gen-scala-server/00-interop/build.sbt

```diff
@@ -3,8 +3,12 @@ organization := "com.lightbend.akka.grpc"
 // For the akka-http snapshot
 resolvers += Resolver.bintrayRepo("akka", "maven")
 
+val grpcVersion = "1.16.1" // checked synced by GrpcVersionSyncCheckPlugin
+
 libraryDependencies ++= Seq(
-  "io.grpc"                  % "grpc-interop-testing"    % "1.14.0"                     % "protobuf",
+  // Excluding grpc-alts works around a complex resolution bug
```

Comment:
So we don't really need grpc-alts on the classpath?

Reply:
Well, it seems not, but I have to admit that I didn't dig in to see what it actually does. It was only used in interop testing, and our interop tests are passing without it.

I'm not sure if we should really merge in this state, or report the issue upstream. As @eed3si9n pointed out, the behavior of sbt/Ivy makes sense, so it's possible that this is actually a bug in Gradle, in the grpc-java build, or both (plus Coursier).